src/components/common/maps/Map.js

```diff
@@ -72,34 +69,32 @@ class SimpleMap extends PureComponent {
   /*
     EVENTS
   */
-  _onClick = ({ x, y, lat, lng, event }) => console.log(x, y, lat, lng, event)
 
-  _onBoundsChange = (center, zoom /* , bounds, marginBounds */) => {
+  /* Parameters { x, y, lat, lng, event } */
+  _onClick = () => {}
+
+  /* Parameters center, zoom, bounds, marginBounds */
+  _onBoundsChange = (center, zoom) => {
     this.setState({ currentCenter: center });
     this.setState({ currentZoom: zoom });
   }
 
-  _onChildClick = (key, childProps) => {
-    console.log(`_onChildClick key: ${key})`);
-    console.log(`childProps: ${childProps})`);
-  }
+  /* Parameters { key, childProps } */
+  _onChildClick = () => {}
```

Comment:
Can you remove these empty functions from here and the props you pass onto the map? No need to have them if you don't use them for anything.

Reply:
I added these in order to show the possible functions to use. but I going to remove this.